File: phm-server/src/main/java/com/phm/model/entity/MedicalItem.java
package com.phm.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import java.math.BigDecimal;

@Data
@TableName("medical_item")
public class MedicalItem {
    @TableId(type = IdType.AUTO)
    private Long id;          // 主键ID
    private String name;      // 项目名称
    private String category;  // 项目分类（如手术/检测等）
    private String workedVariety; // 适用品种（犬、猫等）
    private BigDecimal price; // 项目价格
    private String description; // 详细描述
}

File: phm-ui/src/model/entiy/MedicalItem.ts
// 医疗项目实体类型（对应Java的MedicalItem）
interface MedicalItem {
  id?: number;            // 主键ID（可选，新增时无ID）
  name: string;           // 项目名称（必填）
  category?: string;      // 项目分类（如手术/检测等，可选）
  workedVariety?: string; // 适用品种（犬、猫等，可选）
  price: number;          // 项目价格（必填，对应Java的BigDecimal）
  description?: string;   // 详细描述（可选）
}